learning-content/beginners-guides/evalopensourcevspropietary.md

```diff
@@ -0,0 +1,149 @@
+
+This Beginner’s Guide explores these nuances, outlining common trade-offs, relevant costs, and security and privacy concerns that are most common when evaluating OSS versus proprietary software. The goal of this Beginner’s Guide is to offer practitioners in the social sector, namely M&E professionals, data analysts, software engineers, among others, a summary of potential tradeoffs.These insights were gathered from conversations with social sector professionals, as well as a survey conducted online. Below we provide a summary of our methods and analysis, a high-level summary of our findings, and a framework to evaluate OSS versus proprietary solutions.  
+
+##### Methods
+To understand the potential tradeoffs when evaluating OSS versus proprietary software within the context of data, this Beginner’s Guide adopted a triangulated approach. First, desk research was conducted to establish a knowledge baseline and broadly understand the common tradeoffs between OSS and proprietary solutions. Next, a survey was designed and disseminated via professional networks of program managers, data analysts, M&E professionals, and software developers. The survey(n=24) captured both quantitative and qualitative responses regarding the common constraints and considerations that practitioners take into when evaluating OSS and proprietary software. Using these responses as a guide, semi-structured interviews  were conducted with practitioners (n=6) in the social impact space, including  data scientists (n=1), software engineers (n=3), and monitoring and evaluation professionals (n=3). Triangulating these data with survey responses and desk research, the team conducted descriptive analyses to identify thematic topics and common constraints when evaluating OSS and proprietary software. 
+##### Key terminology 
+In this Beginner’s Guide, OSS is broadly defined as software that is freely distributed and uploaded to an accessible repository that provides source code. More specifically, code, data, or content has to be under an open source license, insofar that the code base is accessible at no cost (see The Open Source Initiative for further outline and definitions). Proprietary software, on the other hand, is copyrighted, such that limits against use, distribution, and/or modification are imposed by the developer. Moreover, along the continuum of software development licenses also exists a business model known as Open Core. That is, Open Core is generally understood as the monetization of commercially produced OSS, proprietary add-ons that provide additional features that come with a commercial license. Examples of Open Core Software include many Service (SaaS) products, like Kafka, a data streaming service under Apache 2.0, Oracle’s MySQL database, Neo4j with the enterprise addition under a commercial license, among many others. To conceptualize the range of differences between OSS, Open Core, and proprietary software, see Figure 1:
```

Comment:
For the line: "Proprietary software, on the other hand, is copyrighted, such that limits against use, distribution, and/or modification are imposed by the developer."

should it be "imposed by the owner"?

Reply:
Revised.